README.md

```diff
@@ -61,3 +61,25 @@ Further documentation: https://docs.vaticle.com/docs/client-api/java
    bazel-bin/com.vaticle.typedb:api.jar
    bazel-bin/pom.xml
    ```
+
+## FAQs
+
+**Q:** I see a large number of Netty and gRPC log messages. How can I disable them?
+
+**A:** 
+1. Create a file in your `resources` path (`src/main/resources` by default in a Maven project) named `logback.xml`.
```

Comment:
```
**A:**

Make sure that Logback is configured properly. You can do so with the following steps:

1. ...
2. ...
```

Reply:
Also, how can we communicate it in the [client docs](https://docs.vaticle.com/docs/client-api/java) as well?